repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/module-setup.sh

```diff
@@ -30,11 +30,17 @@ install() {
     #dracut_install /etc/selinux/*/policy/*
     #dracut_install $(find /etc/selinux/*/contexts)
 
+    # shellcheck disable=SC2154 # Following variables are assigned by dracut
+    { # NOTE: use these new variables instead to avoid the warning
+    local _initdir=$initdir
+    local _moddir=$moddir
+    }
+
     # NOTE: rather remove it then keep it; we really don't want plymouth now
     #       (hopefully forever)
     # remove the plymouth text plugin so we get either graphics or details
-    rm -rf ${initdir}/$(plymouth --get-splash-plugin-path)text.so \
-          ${initdir}/usr/share/plymouth/themes/text/*
+    rm -rf "${_initdir}/$(plymouth --get-splash-plugin-path)text.so" \
+           "${_initdir}/usr/share/plymouth/themes/text"
```

Comment:
This is changed behaviour. We do not want to remove the `text` directory. We want to remove only its content. I do not rember now the reason why not to remove the directory itself however. We could investigate that whether there is a reson for it, but right now I prefer no change that.